projects/components/src/form/form-input/form-input.component.spec.ts

```diff
@@ -12,7 +12,7 @@ import { FormInputComponent, getFormattedDateValue } from './form-input.componen
 export class VcdFormInputWidgetObject extends WidgetObject<FormInputComponent> {
     static tagName = `vcd-form-input`;
 
-    private get inputElement(): HTMLInputElement {
+    get inputElement(): HTMLInputElement {
```

Comment:
Why are we exposing an HTML input element? We typically expose strings, booleans, number